src/models/notificationsettings/toNotificationSettings.ts

```diff
@@ -37,6 +37,22 @@ function shouldNotify(rules: (IPushRule | null | undefined | false)[]): boolean
     return false;
 }
 
+function isMuted(rules: (IPushRule | null | undefined | false)[]): boolean {
+    if (rules.length === 0) {
+        return false;
+    }
+    for (const rule of rules) {
+        if (rule === null || rule === undefined || rule === false || !rule.enabled) {
+            continue;
+        }
+        const actions = NotificationUtils.decodeActions(rule.actions);
+        if (actions !== null && !actions.notify) {
+            return true;
+        }
```

Comment:
I think if a room has a sound/highlight tweak it shouldn't be considered as muted, surely? Sure it's uncommon in push rules but possible.